### PR TITLE
refactor: extract jstree into a webjar

### PR DESCRIPTION
**Pull request by ChristianMurphy**
##### Checklist

-   [x] the [individual contributor license agreement][] is signed
-   [x] commit message follows [commit guidelines][]

##### Description of change

Remove jstree from source code, instead import as webjar.
Apparently the webjar folder is automatically served as part of uPortal. :man_shrugging: :white_check_mark:  
selection_011

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#commit
[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl

### CODE REVIEW

**Comment by ChristianMurphy** [comment]
AppVeyor is failing due to unrelated issue from https://issues.jasig.org/browse/UP-4965

File: uPortal-webapp/src/main/webapp/media/skins/respondr/defaultSkin/skin.xml [hunk 1]
```diff
@@ -39,8 +39,8 @@
   <css included="aggregated">/ResourceServingWebapp/rs/jqueryui/1.10.3/theme/smoothness/jquery-ui-1.10.3-smoothness.min.css</css>
 
   <!-- jsTree (entity-selector) -->
-  <css included="plain" resource="true">/scripts/jstree/style.css</css>
-  <css included="aggregated" resource="true">/scripts/jstree/style.min.css</css>
+  <css included="plain" resource="true">/webjars/jstree/dist/themes/default/style.css</css>
+  <css included="aggregated" resource="true">/webjars/jstree/dist/themes/default/style.min.css</css>
```
Comment (drewwills) [inline uPortal-webapp/src/main/webapp/media/skins/respondr/defaultSkin/skin.xml]: Hold up!

Now that I see it, I'm wondering if these lines should be moved to `common_skin.xml`.

**Comment by drewwills** [comment]
I hope this change works -- it's definitely an improvement.  :+1:

File: uPortal-webapp/src/main/webapp/media/skins/respondr/defaultSkin/skin.xml (hunk 1 above)
Comment (ChristianMurphy) [reply uPortal-webapp/src/main/webapp/media/skins/respondr/defaultSkin/skin.xml]: moved :white_check_mark:

**Pull request merged**